content/vault/v1.18.x/content/docs/upgrading/upgrade-to-1.16.x.mdx

```diff
@@ -18,6 +18,48 @@ Vault 1.15. **Please read carefully**.
 
 ## Important changes
```

Comment:
Please also add a row to the relevant summary table under `/global/partials/important-changes/summary-tables` so this shows up in the change tracker pages